src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -214,6 +215,47 @@ protected function sort_events( $events ) {
 
 	}
 
+	/**
+	 * Creates a HTML link and "fast forward" message to append to the "no events found" message.
+	 *
+	 * @param bool  $canonical         Whether to return the canonical (pretty) version of the URL or not.
+	 * @param array $passthru_vars     An optional array of query variables that should pass thru the method untouched
+	 *                                 in key and value.
+	 * @return string                  The html link and message.
+	 */
+	public function fast_forward( $canonical = false, array $passthru_vars = [] ) {
+		if ( ! empty( $this->context->get( 'tribe-bar-date' ) ) ) {
+			$date = $this->context->get( 'tribe-bar-date' );
+		} else if ( ! empty( $this->context->get( 'event_date' ) ) ) {
```

Comment:
:warning: Usage of ELSE IF is discouraged; use ELSEIF instead

<sub>hash: 3d5792e0fa71bbc1cda133d16192d94b</sub>